src/platform/inprocess/mod.rs

```diff
@@ -261,12 +250,23 @@ impl OsIpcOneShotServer {
         },name.clone()))
     }
 
-    pub fn accept(self) -> Result<(OsIpcReceiver,
-                                    Vec<u8>,
-                                    Vec<OsOpaqueIpcChannel>,
-                                    Vec<OsIpcSharedMemory>),MpscError>
-    {
-        let record = ONE_SHOT_SERVERS.lock().unwrap().get(&self.name).unwrap().clone();
+    pub fn accept(
+        self,
+    ) -> Result<
+        (
+            OsIpcReceiver,
+            Vec<u8>,
+            Vec<OsOpaqueIpcChannel>,
+            Vec<OsIpcSharedMemory>,
+        ),
+        ChannelError,
+    > {
+        let record = ONE_SHOT_SERVERS
+            .lock()
+            .unwrap()
+            .get(&self.name)
+            .unwrap()
+            .clone();
```

Comment:
`MpscError` was changed to `ChannelError`, but other than that, the code is the same. This formatting was produced by rustfmt. The original is kinda messy - misaligned indentation, missing spaces after commas, etc. I can roll back a few formatting changes...

Reply:
I meant just the last line. The function signature reformatting is fine, since you have to touch that anyway -- and the change is a definite improvement there :-)

Reply:
Actually, in this one instance, I actually *do* have a minor qualm about the function signature formatting, which I considered mentioning: with the indentation style change, the tuple would now easily fit on one line -- and I tend to think that would be more readable... But then again, I don't really have experience with this style yet -- maybe the more spacey variant is actually more readable after some getting used to?... Not sure. (Which is why I didn't bring it up originally.)